.github/workflows/publish.yaml

```diff
@@ -61,9 +61,9 @@ jobs:
             poetry publish --repository testpypi
         continue-on-error: false
 
-      - name: Publish dist to real pypi
+      - name: Publish to PyPI
         env:
-            POETRY_PYPI_TOKEN: ${{ secrets.PYPI_KEY }}
+            POETRY_PYPI_TOKEN_PYPI: ${{ secrets.PYPI_KEY }}
```

Comment:
This is the change in env var name according to the Poetry docs: https://python-poetry.org/docs/configuration/#pypi-tokenname